store/sqlstore/session_store.go

```diff
@@ -106,34 +106,28 @@ func (me SqlSessionStore) Get(sessionIdOrToken string) store.StoreChannel {
 	})
 }
 
-func (me SqlSessionStore) GetSessions(userId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var sessions []*model.Session
-
-		tcs := me.Team().GetTeamsForUser(userId)
+func (me SqlSessionStore) GetSessions(userId string) ([]*model.Session, *model.AppError) {
+	var sessions []*model.Session
 
-		if _, err := me.GetReplica().Select(&sessions, "SELECT * FROM Sessions WHERE UserId = :UserId ORDER BY LastActivityAt DESC", map[string]interface{}{"UserId": userId}); err != nil {
-			result.Err = model.NewAppError("SqlSessionStore.GetSessions", "store.sql_session.get_sessions.app_error", nil, err.Error(), http.StatusInternalServerError)
-		} else {
+	if _, err := me.GetReplica().Select(&sessions, "SELECT * FROM Sessions WHERE UserId = :UserId ORDER BY LastActivityAt DESC", map[string]interface{}{"UserId": userId}); err != nil {
+		return nil, model.NewAppError("SqlSessionStore.GetSessions", "store.sql_session.get_sessions.app_error", nil, err.Error(), http.StatusInternalServerError)
+	}
 
-			result.Data = sessions
-		}
+	rtcs := <-me.Team().GetTeamsForUser(userId)	
```

Comment:
You are adding an extra "tab" at the end of this line, which is generating a check-style problem.